### ISSUE TITLE
[Spotify Player] ...

### ISSUE THREAD

**Issue by JunDeer** [raycast/extensions#8862]
### Extension

https://www.raycast.com/mattisssa/spotify-player

#### Error:
```
The API file appears to be corrupted. You may need to reinstall Raycast.


```

### Steps To Reproduce

Just set and change an Hotkey of this shortcut, then it corrupted....

**Comment by sxn** [comment]
hey, thanks for the report. This is an issue with Raycast, not the extension. We're currently addressing it and will deploy a hotfix shortly. Sorry for the hassle 🙏

**Comment by sxn** [comment]
We've just published a hotfix (`1.60.1`) that should address the issue. You can download it from https://www.raycast.com, or run the `Check For Updates` command. ❤️
